src/py/flwr/server/serverapp/app.py

```diff
@@ -69,47 +68,17 @@ def flwr_serverapp() -> None:
     parser.add_argument(
         "--superlink",
         type=str,
-        help="Address of SuperLink's DriverAPI",
-    )
-    parser.add_argument(
-        "--run-once",
-        action="store_true",
-        help="When set, this process will start a single ServerApp "
-        "for a pending Run. If no pending run the process will exit. ",
-    )
-    parser.add_argument(
-        "--flwr-dir",
-        default=None,
-        help="""The path containing installed Flower Apps.
-    By default, this value is equal to:
-
-        - `$FLWR_HOME/` if `$FLWR_HOME` is defined
-        - `$XDG_DATA_HOME/.flwr/` if `$XDG_DATA_HOME` is defined
-        - `$HOME/.flwr/` in all other cases
-    """,
-    )
-    parser.add_argument(
-        "--insecure",
-        action="store_true",
-        help="Run the server without HTTPS, regardless of whether certificate "
-        "paths are provided. By default, the server runs with HTTPS enabled. "
-        "Use this flag only if you understand the risks.",
-    )
-    parser.add_argument(
-        "--root-certificates",
-        metavar="ROOT_CERT",
-        type=str,
-        help="Specifies the path to the PEM-encoded root certificate file for "
-        "establishing secure HTTPS connections.",
+        help="Address of SuperLink's ServerAppIo gRPC servicer",
```

Comment:
@danieljanes @jafermarq Should we call it `xxx API`, `xxx API server`, or `xxx gRPC servicer`? I think we also need to update the help message for `flwr-clientapp` after we decide on this.
```suggestion
        help="Address of SuperLink's ServerAppIo API server",
```

Reply:
`xxx API` sounds good to me

Reply:
I agree, I'll switch to `xxx API`.